aria-practices.html

```diff
@@ -6279,6 +6279,38 @@ <h3>Indicating sort order with <code>aria-sort</code></h3>
 
   </section>
 
+  <section id="accessibility_tree">
+    <h2>Accessibility tree</h2>
+    <p>Browsers use a so-called accessibility tree to communicate the representation of web content with assistive technologies. See the <a href="https://w3c.github.io/core-aam/#dfn-accessibility-tree">Core Accessibility API Mappings</a> specification for further information about accessibility trees.</p>
```

Comment:
I would prefer not to use the  phrase "so-called" because it typically means "inappropriately named".
Maybe something like:
> Browsers construct an accessibility tree to communicate the representation of web content with assistive technologies.

Reply:
Thanks, I didn't know "so-called" had that meaning.